### ISSUE TITLE
When i open Deco android emulator crash

### ISSUE THREAD

**Issue by scerelli** [decosoftware/deco-ide#62]
I just set up a project using ignite for react-native but deco does not seem to work well using genymotion emulator.

This is what happens:

1) I start genymotion emulator (using sdk installed in `$HOME/Library`)
2) `react-native run-android` inside the project folder
3) I open Deco
4) System crash

The moment i open deco watchman loses its connection and everything crash
```
[sane] Warning: Lost connection to watchman, reconnecting..
[sane] Warning: Lost connection to watchman, reconnecting..
2016-05-31T15:06:49,203: [cli] failed to identify PDU: fill_buffer: EOF

2016-05-31T15:06:49,204: [cli] unable to talk to your watchman on /usr/local/Cellar/watchman/HEAD/var/run/watchman/voodo-state/sock! (Undefined error: 0)

2016-05-31T15:06:49,203: [cli] failed to identify PDU: fill_buffer: EOF
2016-05-31T15:06:49,205: [cli] unable to talk to your watchman on /usr/local/Cellar/watchman/HEAD/var/run/watchman/voodo-state/sock! (Undefined error: 0)

Watchman:  watchman--no-pretty get-sockname returned with exit code 1 2016-05-31T15:06:49,203: [cli] failed to identify PDU: fill_buffer: EOF
2016-05-31T15:06:49,204: [cli] unable to talk to your watchman on /usr/local/Cellar/watchman/HEAD/var/run/watchman/voodo-state/sock! (Undefined error: 0)

 ERROR  watchman--no-pretty get-sockname returned with exit code 1 2016-05-31T15:06:49,203: [cli] failed to identify PDU: fill_buffer: EOF
2016-05-31T15:06:49,204: [cli] unable to talk to your watchman on /usr/local/Cellar/watchman/HEAD/var/run/watchman/voodo-state/sock! (Undefined error: 0)

Error: watchman--no-pretty get-sockname returned with exit code 1 2016-05-31T15:06:49,203: [cli] failed to identify PDU: fill_buffer: EOF
2016-05-31T15:06:49,204: [cli] unable to talk to your watchman on /usr/local/Cellar/watchman/HEAD/var/run/watchman/voodo-state/sock! (Undefined error: 0)

    at ChildProcess.<anonymous> (/Users/voodo/Development/tbpclient/node_modules/fb-watchman/index.js:198:18)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:850:16)
    at Socket.<anonymous> (internal/child_process.js:323:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:492:12)
```

**Comment by dchen** [comment]
This will likely be resolved with issue #39 to add an option for Deco not to launch watchman when running it externally.

When you say "System crash", is Deco exiting, also the emulator and the packager running in Terminal is ending as well?

**Comment by gdowens** [comment]
Watchman fix should solve these issues — reopen if issue persists on next release.
